traveler_core_kit/src/androidTest/java/com/guestlogix/travelercorekit/TravelerTest.java

```diff
@@ -62,4 +60,81 @@ public void onFlightSearchError(Error error) {
         Assert.assertEquals("Error(s) occurred while querying API for flight 'AC100'!\n" +
                 "See logcat for more info\n", 0, errorList.size());
     }
+
+    @Test
+    public void catalogueTest() throws Exception {
```

Comment:
incorrect spelling

Reply:
Also in commit messages, (just saying for the future, no need to reedit the messages, but the method needs to read correctly)

Reply:
heh, didn't catch my canadian spelling there. fixed.